cdap-unit-test/src/main/java/co/cask/cdap/test/UnitTestManager.java

```diff
@@ -86,6 +91,24 @@
 public class UnitTestManager implements TestManager {
 
   private static final Gson GSON = new Gson();
+  private static final ClassAcceptor CLASS_ACCEPTOR = new ClassAcceptor() {
```

Comment:
For now I keep them separated in case they need to behaves differently (on unit-test all classes are from system classpath vs integrated test may not).